ultimate_cron.module

```diff
@@ -217,292 +147,7 @@ function ultimate_cron_prepare_job($name, $hook, $job = NULL) {
- *
- * Set job status and log it.
- *
- * @param mixed $object
- *   Name of job or UltimateCronJob object.
- * @param boolean $status
- *   The status to set (TRUE = disabled).
- */
-function ultimate_cron_job_set_status($object, $status) {
-  if (is_object($object)) {
-    $name = $object->name;
-    $job = $object;
-  }
-  else {
-    $name = $object;
-    $job = ultimate_cron_job_load($name);
-  }
-
-  if (empty($object->dont_log)) {
-    $log_entry = $job->startLog(uniqid($job->id(), TRUE), 'modification', ULTIMATE_CRON_LOG_TYPE_ADMIN);
-    $log_entry->log($job->id(), 'Job @status by ' . $log_entry->formatUser(), array(
-      '@status' => $status ? t('disabled') : t('enabled'),
-    ), RfcLogLevel::INFO);
-    $log_entry->finish();
-  }
-
-  variable_set('default_ultimate_cron_job_' . $name, $status ? TRUE : FALSE);
-}
-
-/**
- * CTools Export delete callback.
- *
- * Revert a job and log it.
- *
- * @param mixed $object
- *   Name of job or UltimateCronJob object.
- */
-function ultimate_cron_job_delete($object) {
-  $table = 'ultimate_cron_job';
-  $schema = ctools_export_get_schema($table);
-  $export = $schema['export'];
-
-  // If we were sent an object, get the export key from it. Otherwise
-  // assume we were sent the export key.
-  if (!is_object($object)) {
-    $name = $object;
-    if (!($object = ultimate_cron_job_load($name))) {
-      throw new Exception(t('Unable to revert unknown job: @name', array(
-        '@name' => $name,
-      )));
-    }
-  }
-
-  db_delete($table)
-    ->condition($export['key'], $object->{$export['key']})
-    ->execute();
-
-  // @TODO: Move to CronJob postDelete() and delete function
-  if (empty($object->dont_log)) {
-    $log = $object->startLog(uniqid($object->name, TRUE), 'modification', ULTIMATE_CRON_LOG_TYPE_ADMIN);
-    $log->log($object->name, 'Job reverted by ' . $log->formatUser(), array(), RfcLogLevel::INFO);
-    $log->finish();
-  }
-}
-
-// ---------- HOOKS ----------
-/**
- * Implements hook_hook_info().
- */
-function ultimate_cron_hook_info() {
-  $hooks = array();
-  $hooks['background_process_shutdown'] = array(
-    'group' => 'background_process_legacy',
-  );
-  return $hooks;
-}
-
-/**
- * Implements hook_init().
- *
- * Make sure we have the proper "last run" of cron in global $conf
- * for maximum compatibility with core.
- *
- * @todo: Port this.
- */
-function ultimate_cron_init() {
-  if (!variable_get('ultimate_cron_bypass_transactional_safe_connection')) {
-    $info = Database::getConnectionInfo('default');
-    Database::addConnectionInfo('default', 'ultimate_cron', $info['default']);
-  }
-  $GLOBALS['ultimate_cron_original_session_saving'] = drupal_save_session();
-  $GLOBALS['ultimate_cron_original_user'] = $GLOBALS['user'];
-
-  _ultimate_cron_variable_load('cron_last');
-}
-
-/**
- * Implements hook_help().
- *
- */
-function ultimate_cron_help($route_name, \Drupal\Core\Routing\RouteMatchInterface $route_match) {
-  switch ($route_name) {
```

Comment:
You can't just remove hooks, they're not called as functions, but they still exist. And if a hook doesn't exist anymore, we need a plan to update it, as mnetioned above.